### DISCUSSION TITLE
Incomplete recognition

### DISCUSSION THREAD

**Discussion by lxfzerolearning** [remarkjs/remark#1029] [General]
Markdown parts of the syntax render unsuccessful, such as unordered lists, strikeouts, etc., what should be done

**Comment by ChristianMurphy** [comment]
remark by default supports everything in commonmark https://commonmark.org/help/
Some features like strike through are part of another standard GitHub flavored markdown, which can be enabled with https://github.com/remarkjs/remark-gfm

If you have further questions on how to enable syntax features, consider using a sandbox
| description | codesandbox | stackblitz |
| - | - | - |
| remark only (for markdown to markdown) | [codesandbox](https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/remark-with-parcel) | [stackblitz](https://stackblitz.com/github/remarkjs/.github/tree/main/sandbox-templates/remark-with-parcel) |
| remark and rehype (for markdown to html) | [codesandbox](https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/remark-rehype-with-parcel) | [stackblitz](https://stackblitz.com/github/remarkjs/.github/tree/main/sandbox-templates/remark-rehype-with-parcel) |
| react-markdown | [codesandbox](https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app) | [stackblitz](https://stackblitz.com/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app) |


to show what syntax you are trying and how you have configured remark.
